app/src/main/java/com/sdpteam/connectout/event/EventFirebaseDataSource.java

```diff
@@ -83,6 +85,7 @@ public void onComplete(DatabaseError databaseError, boolean committed, DataSnaps
      * @return (CompletableFuture < Boolean >): completes to true if participant has joined the event.
      */
     public CompletableFuture<Boolean> joinEvent(String eventId, String participantId) {
+        new ProfileFirebaseDataSource().registerToEvent(new RegisteredEvent(eventId), participantId);
```

Comment:
Good idea, so the code Mikael did is still working 👍

Reply:
thanks, didn't have time to refactor everything